protostar/cheatable_starknet/controllers/transaction_revert_exception.py

```diff
@@ -1,7 +1,26 @@
+import ast
 from dataclasses import dataclass
 
+from starkware.cairo.lang.compiler.test_utils import short_string_to_felt
+
+from protostar.protostar_exception import ProtostarException
+
 
 @dataclass
 class TransactionRevertException(Exception):
     message: str
     raw_ex: Exception
+
+    def get_panic_data(self) -> list[int]:
+        msg = self.message.replace("Execution was reverted; failure reason:", "")
+        if msg.startswith(" [") and msg.endswith("]."):
+            msg = msg.replace(" [", "[")
+            msg = msg.replace("].", "]")
+
+            parsed_message_felts = ast.literal_eval(msg)
```

Comment:
why is this needed?

Reply:
we get a `[1, 2, 3, 4]` list at the end, this one basically safely parses it into python expression